### PR TITLE
Fix a bunch of stuff

### PR DESCRIPTION
**Pull request by ivarnakken**
# Description

Picked up by this crawler:

image

The content authors object is used for comments to highlight the creator
/ author of the content in the comment section. But this would ultimately
break the event page when it would try to read id off of null.

---

[Fix misalignment of anonymous attendance status](https://github.com/webkom/lego-webapp/commit/d038eba8bc16ac29b1c499d2f860c68356b37b0b) 

It was the tooltip that caused the misalignment, so just repositioning
it fixed the bug. A maximum width was also added to prevent tooltips
with a lot of content text from just becoming one very long line.

Some minor additional styling was added by utilizing the button component,
and disabling it when the user is not authenticated.

---

[Type the event page more thoroughly](https://github.com/webkom/lego-webapp/commit/1380d5febfc298f6e40afe4c6039d911ffdd4e86) 

A lot of the store types were wrong.

# Result

<table>
	<tr>
		<td>Before</td>
		<td>After</td>
	</tr>
	<tr>
		<td>
			image
		</td>
		<td>
			image
		</td>
	</tr>
</table>

# Testing

- [x] I have thoroughly tested my changes.

Highlighting content authors still works:
image

### CODE REVIEW

**Review by LudvigHz — APPROVED** [review]

**Comment by ivarnakken** [comment]
This PR ended up fixing some other stuff as well heh sowwy

File: app/components/Image/ProfilePicture.tsx [hunk 1]
```diff
@@ -1,28 +1,27 @@
...
+  size: number;
+  className?: string;
+  alt?: string;
+} & Omit<ComponentProps<typeof CircularPicture>, 'src' | 'alt'>;
 
 const ProfilePicture = ({
   user,
   size = 100,
   className,
+  alt = `${user.username}'s profile picture`,
```
Comment (LudvigHz) [inline app/components/Image/ProfilePicture.tsx:19]: Why bother with the alt here when this is the only one that makes sense? Just remove the alt prop alltogether and use this one always

**Review by LudvigHz — APPROVED** [review]
LGTM

**Review by eikhr — APPROVED** [review]

File: app/components/Image/ProfilePicture.tsx (hunk 1 above)
Comment (ivarnakken) [reply app/components/Image/ProfilePicture.tsx:19]: Idk. I just thought it was nice to let users override _if_ they wanted to - but yes never should heh

**Comment by LudvigHz** [comment]
@ivarnakken Can I/you merge this now? I need my branch without this flaky test :grimacing:

**Pull request merged**